modules/appeals_api/app/sidekiq/appeals_api/appeal_received_job.rb

```diff
@@ -9,152 +9,71 @@ class AppealReceivedJob
+        raise "Missing arguments in #{self.class.name}: Received #{argument_list.join(', ')}"
       end
-    end
 
-    def template_id(template)
-      t = claimant? ? "#{template}_claimant" : template
-      template_id = Settings.vanotify.services.lighthouse.template_id.public_send(t)
+      appeal = appeal_class_str.constantize.find(appeal_id)
 
-      [t, template_id]
-    end
-
-    def personalisation
-      p = { 'date_submitted' => date_submitted }
-      if claimant?
-        p['first_name'] = opts['claimant_first_name']
-        p['veterans_name'] = opts['first_name']
-      else
-        p['first_name'] = opts['first_name']
+      unless appeal.form_data.present? && appeal.auth_headers.present?
+        raise "Missing PII for #{appeal_class_str} #{appeal_id}"
       end
-      { personalisation: p }
-    end
-
-    def log_error(guid, type)
-      Rails.logger.error "No lookup value present for AppealsApi::AppealReceived notification #{type} - GUID: #{guid}"
-    end
 
-    def guid
-      opts['guid']
-    end
-
-    def date_submitted
-      @date_submitted ||= DateTime.iso8601(opts['date_submitted']).strftime('%B %d, %Y')
-    end
-
-    def valid_email_identifier?
-      if claimant?
-        opts['claimant_email'].present?
+      vanotify_service.send_email(vanotify_args(appeal, date_submitted_str))
+
+      StatsD.increment(
+        STATSD_CLAIMANT_EMAIL_SENT,
+        tags: {
+          appeal_type: appeal.class.name.demodulize.scan(/\p{Upper}/).map(&:downcase).join,
+          claimant_type: appeal.claimant.signing_appellant? ? 'non-veteran' : 'veteran'
+        }
+      )
+    end
+
+    def vanotify_args(appeal, date_submitted_str)
+      common_info = {
+        template_id: appeal_template_id(appeal),
+        date_submitted: DateTime.iso8601(date_submitted_str).strftime('%B %d, %Y')
+      }
+
+      if appeal.claimant.signing_appellant?
+        {
+          **common_info,
+          email_address: appeal.claimant.email,
+          personalisation: { first_name: appeal.claimant.first_name, veterans_name: appeal.veteran.first_name }
+        }
       else
-        required_email_identifier_keys.all? { |k| opts.dig('email_identifier', k).present? }
-      end
-    end
+        contact_info = if appeal.email_identifier[:id_type] == 'email'
+                         { email_address: appeal.email_identifier[:id_value] }
+                       else
+                         { recipient_identifier: appeal.email_identifier }
+                       end
 
-    def claimant?
-      opts['claimant_first_name'].present? || opts['claimant_email'].present?
-    end
-
-    def claimant_type
-      claimant? ? 'non-veteran' : 'veteran'
+        { **common_info, **contact_info, personalisation: { first_name: appeal.veteran.first_name } }
+      end
+    rescue Date::Error
+      raise "Invalid date format for #{self.class.name}: '#{date_submitted_str}' must be in iso8601 format"
```

Comment:
Previously, this job would just log errors, but I chose to raise them instead so that it would be more clear to us if we try to send an email under circumstances that we shouldn't (when there is insufficient or incorrect data). Open to reconsidering this for sure.